boards/arm/rp2040/common/src/rp2040_bmp180.c

```diff
@@ -0,0 +1,109 @@
+ * License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ ****************************************************************************/
+
+/****************************************************************************
+ * Included Files
+ ****************************************************************************/
+
+#include <nuttx/config.h>
+
+#include <stdio.h>
+
+#include <nuttx/arch.h>
+#include <nuttx/sensors/bmp180.h>
+#include <nuttx/i2c/i2c_master.h>
+
+#include "rp2040_i2c.h"
+#include "rp2040_bmp180.h"
+
+/****************************************************************************
+ * Pre-processor Definitions
+ ****************************************************************************/
+
+/****************************************************************************
+ * Private Types
+ ****************************************************************************/
+
+/****************************************************************************
+ * Private Function Prototypes
+ ****************************************************************************/
+
+/****************************************************************************
+ * Private Data
+ ****************************************************************************/
+
+/****************************************************************************
+ * Public Data
+ ****************************************************************************/
+
+/****************************************************************************
+ * Private Functions
+ ****************************************************************************/
+
+/****************************************************************************
+ * Public Functions
+ ****************************************************************************/
+
+/****************************************************************************
+ * Name: board_bmp180_initialize
+ *
+ * Description:
+ *   Initialize and register the BMP180 Pressure Sensor driver.
+ *
+ * Input Parameters:
+ *   devno - The device number, used to build the device path as /dev/pressN
+ *   busno - The I2C bus number
+ *
+ * Returned Value:
+ *   Zero (OK) on success; a negated errno value on failure.
+ *
+ ****************************************************************************/
+
+int board_bmp180_initialize(int busno)
+{
+  struct i2c_master_s *i2c;
+  char devpath[12];
+  int ret;
+  const int devno = 0;
+
+  sninfo("Initializing BMP180!\n");
+
+  /* Initialize BMP180 */
+
+  i2c = rp2040_i2cbus_initialize(busno);
+
```

Comment:
There is not need for this empty between "i2c = rp2040..." and "if (i2c)", please the NuttX Coding Standard, in the section "4.8 Use of goto" there are many examples showing that there is not need for this empty line. It seams that someone included it in the code and now it is reproducing fast. :-)

Reply:
Hi @phfbertoleti, could you perform this simple test?
Please, remove this blank line and run `./tools/checkpatch.sh -f boards/arm/rp2040/common/src/rp2040_bmp180.c`
If NxStyle indeed reports this issue, I'd say it is an issue with the verification.

Reply:
@gustavonihei , I've checksed as you requested and an error is generated:

pedro@pedro-270E5G-270E5U:~/nuttxspace/fork_repo/incubator-nuttx$ ./tools/checkpatch.sh -f boards/arm/rp2040/common/src/rp2040_bmp180.c 
/home/pedro/nuttxspace/fork_repo/incubator-nuttx/boards/arm/rp2040/common/src/rp2040_bmp180.c:88:1: error: Missing blank line after comment

Reply:
@gustavonihei in this case, I should maintain the code as it is (once current verification require this blank like), right?

Reply:
@gustavonihei , it seems verification considers this blank line required. 

pedro@pedro-270E5G-270E5U:~/nuttxspace/fork_repo/incubator-nuttx$ ./tools/checkpatch.sh -f boards/arm/rp2040/common/src/rp2040_bmp180.c 
/home/pedro/nuttxspace/fork_repo/incubator-nuttx/boards/arm/rp2040/common/src/rp2040_bmp180.c:88:1: error: Missing blank line after comment

Reply:
It seems you've removed the wrong blank line. Actually @acassis suggestion was for removing line 91. Line 88 is indeed required.

Reply:
@gustavonihei and @acassis, sorry, I think I've understood it wrong. I just removed tjhe blankline between rp2040_i2cbus_initialize and futher if command.